jbpm-wb-common/jbpm-wb-common-client/src/main/resources/org/jbpm/workbench/common/client/resources/i18n/Constants.properties

```diff
@@ -31,3 +31,10 @@ LastHours=Last {0} hours
 LastDays=Last {0} days
 Custom=Custom
 Home=Home
+Process_Definitions=Process Definitions
+Process_Instances=Process Instances
+Tasks_List=Tasks List
```

Comment:
Task List

Reply:
@cristianonicolai you mean modify everywhere 'Tasks List' -> 'Task List' 
I mean create a CommonConstants with this values and use that from the different lists and show case?